extensions/functions_aggregate_generic.yaml

```diff
@@ -5,22 +5,21 @@ aggregate_functions:
     description: Count a set of values
     impls:
       - args:
-          - name: overflow
-            options: [SILENT, SATURATE, ERROR]
-            required: false
           - name: x
             value: any
+        options:
+          - name: overflow
+            values: [SILENT, SATURATE, ERROR]
         nullability: DECLARED_OUTPUT
         decomposable: MANY
         intermediate: i64
         return: i64
   - name: "count"
     description: "Count a set of records (not field referenced)"
     impls:
-      - args:
+      - options:
```

Comment:
did you consider making these maps instead of arrays? That way the name is more formally required purely by structure.

Reply:
That's a good idea and solves my other naming problem.  I'll give that a try.

Reply:
Done.